client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+
+export default React.createClass( {
+	displayName: 'BluehostStoreStep',
+
+	redirectToPartner() {
+		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+		} else {
+			window.location.href = 'https://www.bluehost.com/wordpress';
+		}
+	},
+
+	getPrice() {
+		return ( 'bluehost' === abtest( 'signupStoreBenchmarking' ) ) ? '$3.95' : '$12.95';
+	},
+
+	render() {
+		return (
+			<div>
+				<StepHeader
+					headerText={ this.translate( 'Create a WordPress Store' ) }
+					subHeaderText={ this.translate( 'Our partners at BlueHost and WooCommerce are here for you' ) }
```

Comment:
We're also trying not to introduce more code with the implicit translation mixing. Instead, we are wrapping our components in `localize` which provides a translation function as a prop:

```js
import { localize } from 'i18n-calypso';

export const BluehostStoreStep = props => {
	const { translate } = props;

	return <div>{ translate( 'Create a WordPress Store' ) }</div>
};

export default localize( BluehostStoreStep );
```